Tasks/ServiceFabricComposeDeploy/ServiceFabricComposeDeploy.ps1

```diff
@@ -163,48 +163,102 @@ try {
         $getStatusParameters['TimeoutSec'] = $getStatusTimeoutSec
     }
 
+    $upgrading = $false
     $existingApplication = Get-ServiceFabricComposeApplicationStatusHelper -ApiVersion $apiVersion -GetStatusParameters $getStatusParameters
     if ($existingApplication -ne $null)
     {
-        Write-Host (Get-VstsLocString -Key RemovingApplication -ArgumentList $applicationName)
-
-        Remove-ServiceFabricComposeApplicationHelper -ApiVersion $apiVersion -RemoveParameters $removeParameters
-        do
+        if ($apiVersion -eq "2.8")
         {
-            Write-Host (Get-VstsLocString -Key CurrentStatus -ArgumentList $existingApplication.Status)
-            Start-Sleep -Seconds 3
-            $existingApplication = Get-ServiceFabricComposeApplicationStatusHelper -ApiVersion $apiVersion -GetStatusParameters $getStatusParameters
+            $upgrading = $true
```

Comment:
Should a new task input be added which specifies whether to upgrade or not? We should be consistent with Deploy task.

Reply:
The original task has suffered from a great deal of over complexity that has led to user confusion. I'd prefer to wait for user feedback requesting a flag rather then speculatively adding flags just in case.
On top of that the underlying platform feature remains in preview with preiodic breaking changes so I'd like to keep the options focused to reduce the maintence cost.